src/connection_manager.c

```diff
@@ -356,6 +363,11 @@ static qd_error_t load_server_config(qd_dispatch_t *qd, qd_server_config_t *conf
     config->policy_vhost         = qd_entity_opt_string(entity, "policyVhost", 0);    CHECK();
     config->conn_props           = qd_entity_opt_map(entity, "openProperties");       CHECK();
 
+    if (strcmp(config->role, "inter-router") == 0) {
+        config->data_connection_count = qd_entity_opt_long(entity, "dataConnectionCount", 1); CHECK();
```

Comment:
nit: validate data connection count value?  Probably should fail if zero - definitely fail if < 0.